app/components/layout/TabbedPage.js

```diff
@@ -122,16 +122,15 @@ class TabbedPage extends React.Component {
     );
 
     const tabContents = this.props.uiAnimations ? this.animatedStyles() : this.staticStyles();
-
     return (
       <div className="tabbed-page">
         <div className="tabbed-page-header">
           {header}
           <Switch>{headers}</Switch>
-          <RoutedTabsHeader tabs={tabHeaders} />
+          <RoutedTabsHeader tabs={tabHeaders} noCaret={this.props.noCaret}/>
         </div>
 
-        <div className="tabbed-page-body">
+        <div className={"tabbed-page-body" + (this.props.governancePage ? " governance" : "")}>
```

Comment:
Hmm, I'd say that instead of a "governancePage" props this should either be a generic `className` prop appended to this divs classes or a "AltCaret" (or "HightlightCaret", "FullCaret", "NoCaret" or something like that) so that this isn't just a governance page thing.

Reply:
Ah good point, will make the adjustments